js-miniapp-sample/src/services/user/actions.js

```diff
@@ -0,0 +1,86 @@
+import {
+  REQUEST_USER_NAME_SUCCESS,
+  REQUEST_USER_NAME_FAILURE,
+  REQUEST_PROFILE_PHOTO_SUCCESS,
+  REQUEST_PROFILE_PHOTO_FAILURE,
+  REQUEST_CONTACT_LIST_SUCCESS,
+  REQUEST_CONTACT_LIST_FAILURE,
+} from './types';
+
+type UserNameSuccessAction = { type: String, userName: string };
+type ProfilePhotoSuccessAction = { type: String, url: string };
+type ContactListSuccessAction = { type: String, contacts: string[] };
+
+const requestUserName = (): Function => {
```

Comment:
Is this method for requesting the username after the permission granted ?

Reply:
Yeah, I added this so it's ready for when we need it later.